recipe/build.sh

```diff
@@ -1,18 +1,37 @@
-#!/usr/bin/env bash
+#!/bin/bash -e
 
 mkdir build
 cd build
-if [[ ${target_platform} == linux-ppc64le ]]; then
-  cmake -DCMAKE_INSTALL_PREFIX=${PREFIX} \
-        -DCMAKE_PREFIX_PATH=${PREFIX} \
-        -DENABLE_BOOST=ON \
-        -DENABLE_COVERAGE=OFF \
-        -DENABLE_LIBCPP=OFF \
-        -DENABLE_SSE=OFF \
-        ../libhdfs3
+
+CMAKE_OPTS=""
+
+case ${target_platform} in
+  linux-ppc64le)
+    CMAKE_OPTS="
+      -DENABLE_BOOST=ON \
+      -DENABLE_COVERAGE=OFF
+      -DENABLE_LIBCPP=OFF
+      -DENABLE_SSE=OFF
+    "
+    ;;
+  linux-aarch64)
+    CMAKE_OPTS="
+      -DENABLE_SSE=OFF
+    "
+    ;;
+esac
+
+if [ -n "$CMAKE_OPTS" ]; then
+  cmake \
+    -DCMAKE_INSTALL_PREFIX=${PREFIX} \
+    -DCMAKE_PREFIX_PATH=${PREFIX} \
+    $CMAKE_OPTS \
```

Comment:
use ${CMAKE_OPTS} please

Reply:
Ok, sure. I also went over quoting and other variable references again.